dynamic-demo-plugin/console-extensions.json

```diff
@@ -319,6 +319,16 @@
       "component": { "$codeRef": "overviewDetailItem.default" }
     }
   },
+  {
```

Comment:
shall we remove `console.dashboards/overview/detail/item"` code at line 316 ~ line 319?